package.json

```diff
@@ -134,7 +134,6 @@
     "@types/compression": "^1.7.2",
     "@types/connect-flash": "0.0.37",
     "@types/connect-redis": "^0.0.18",
-    "@types/cookie-session": "^2.0.44",
```

Comment:
`cookie-session` was removed previously, but the associated types were not. `cookie-session` is not used, `express-session` is used instead